targets.go

```diff
@@ -72,6 +72,10 @@ func ArgoCDDiff() error {
 	if err != nil {
 		return err
 	}
+	err = validateKyvernoPolicies(apps)
```

Comment:
can we not wrap this in a separate target, like `KubeConform()` (line 34)

Reply:
wrapping this under kubeconform cause validation errors as the parameter passed is not defined there.